src/components/table.js

```diff
@@ -259,37 +277,52 @@ FlexTable.propTypes = {
   headerHeight: PropTypes.number,
   rowHeight: PropTypes.number,
   styleHeader: PropTypes.func,
-  styleCell: PropTypes.func
+  styleCell: PropTypes.func,
+  tableName: PropTypes.string,
+  sort: PropTypes.shape({
+    field: PropTypes.string,
+    direction: PropTypes.string
+  })
 }
 
 /**
  * A basic table with a header and flexible column widths. Intended for small amounts of data,
  * since it does not provide scrolling. See FlexTable for prop types.
  */
-export const SimpleFlexTable = ({ columns, rowCount, noContentMessage, hoverHighlight }) => {
-  return h(Fragment, [
-    div({ style: { height: 48, display: 'flex' } }, [
-      _.map(([i, { size, headerRenderer }]) => {
+export const SimpleFlexTable = ({ columns, rowCount, noContentMessage, hoverHighlight, tableName = 'data table', sort = null }) => {
+  return div({
+    role: 'table',
+    'aria-label': tableName
+  }, [
+    div({
+      style: { height: 48, display: 'flex' },
+      role: 'row'
+    }, [
+      _.map(([i, { size, headerRenderer, field }]) => {
         return div({
           key: i,
-          style: { ...styles.flexCell(size), ...styles.header(i * 1, columns.length) }
-        }, [headerRenderer()])
+          style: { ...styles.flexCell(size), ...styles.header(i * 1, columns.length) },
+          role: 'columnheader',
+          'aria-sort': !sort || !field ? null : sort.field === field ? sort.direction === 'asc' ? 'ascending' : 'descending' : 'none'
```

Comment:
Is there a way to extract this out into a variable? I see it is also used in other tables

Reply:
Hmm. I could calculate it each time within the renderer function and then just use the value here, if that would be preferred. I suppose I could also make this a shared function for all 3 tables, but it would basically be a one-liner.  Probably shouldn't memoize it, though, since the whole point of these virtual tables is to not have the entire thing in memory at a time...

What I would ideally prefer to do is extract it from the `Sortable` renderer nested inside the results from the `headerRenderer()` call. See comments above about avoiding duplication...  I'm just not sure whether it's good practice to introspect hyperscript objects. If you think that's ok, I can remove the `field` property here entirely.

Reply:
I would probably extract it into a function, sort of like:
`(sort, field) => !sort || !field ? null : sort.field === field ? sort.direction === 'asc' ? 'ascending' : 'descending' : 'none'`, and then each place this is duplicated, we can just call that function. I think it could make things a little more readable.